cmd/awscognitouserpoolsource/kodata/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/master
```

Comment:
Those should be symlinks. You can copy these from another cmd directory using the `-P` option:
```console
$ cp -rP cmd/aws-event-sources-controller/kodata cmd/awscognitouserpoolsource/
```

Reply:
missed this, thanks